mmv1/third_party/terraform/website/docs/r/eventarc_trigger.html.markdown

```diff
@@ -19,11 +19,13 @@ An event trigger sends messages to the event receiver service deployed on Cloud
 resource "google_eventarc_trigger" "trigger" {
   name = "trigger"
   location = "us-central1"
+  // Note: matching_criteria is named event_filters in the v1 API
```

Comment:
```suggestion
  # note: matching_criteria in gcloud
```

Wdyt of this for both places?

Doesn't Terraform use `#` for comments?

Reply:
Ah, I guess `gcloud` support doesn't exist yet. Maybe `also known as matching_criteria` or something. I feel like a terse, informal message will work best here (with a more complete one below the fold)